MomsSpaghetti/Pages/Index.cshtml.cs

```diff
@@ -160,43 +160,45 @@ public ActionResult OnGetLiked()
             return new JsonResult(response);
         }
 
-        public  ActionResult OnPostSend()
+        public ActionResult OnPostSend()
         {
 
 
             // JObject returnTemp = null;
             var likedRecipes = new List<JObject>();
             string requestBody;
-            string likedRecipesJSON="";
-                MemoryStream stream = new MemoryStream();
-                Request.Body.CopyTo(stream);
-                stream.Position = 0;
-                using (StreamReader reader = new StreamReader(stream))
+            string likedRecipesJSON = "";
+            MemoryStream stream = new MemoryStream();
+            Request.Body.CopyTo(stream);
+            stream.Position = 0;
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                requestBody = reader.ReadToEnd();
+                if (requestBody.Length > 0)
                 {
-                     requestBody = reader.ReadToEnd();
-                    if (requestBody.Length > 0)
+                    var obj = JObject.Parse(requestBody);
+                    if (obj != null)
                     {
-                        var obj = JObject.Parse(requestBody);
-                        if (obj != null)
-                        {
                         // var temp = Like.Likes;
                         // var  myList = new List<Object>();
                         var cachedResult = _cache.Get<List<JObject>>("LikedRecipes");
-                        if (Like.Likes == null && cachedResult == null) {
+                        if (Like.Likes == null && cachedResult == null)
+                        {
                             Like.Likes = new List<JObject>();
                             Like.Likes.Add(obj);
                             _cache.Set<List<JObject>>("LikedRecipes", Like.Likes);
                         }
-                        else {
+                        else
+                        {
                             Like.Likes = _cache.Get<List<JObject>>("LikedRecipes");
                             Like.Likes.Add(obj);
                             _cache.Set<List<JObject>>("LikedRecipes", Like.Likes);
                         }
-                       
-                        
-                       // ViewData["LikedRecipes"] = _cache.Get<List<JObject>>("LikedRecipes");
-                       likedRecipes = _cache.Get<List<JObject>>("LikedRecipes");
-                        
+
+
+                        // ViewData["LikedRecipes"] = _cache.Get<List<JObject>>("LikedRecipes");
+                        likedRecipes = _cache.Get<List<JObject>>("LikedRecipes");
+
```

Comment:
For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.